zerver/lib/notifications.py

```diff
@@ -110,6 +109,18 @@ def relative_to_full_url(base_url, content):
 
     return content
 
+def fix_emojis(content, base_url):
+    # type: (Text, Text) -> Text
+    # Convert the emoji spans to img tags.
+    content = re.sub(
+        r'<span class=\"emoji emoji-(\S+)\" title=\"([^\"]+)\">(\S+)</span>',
+        r'<img src="' + base_url + r'/static/generated/emoji/images-google-64/\1.png" ' +
+        r'title="\2" alt="\3" style="height: 20px;">',
+        content)
+    content = content.replace(' class="emoji"', ' style="height: 20px;"')
+
+    return content
```

Comment:
Just a comment: I think we may want to eventually migrate this code path to use xml.ElementTree for doing the rewriting; that would likely be a lot more robust than the regular expressions we've been doing.

But it's not a regression, so OK for now :)

Reply:
Yeah I was also thinking of replacing these regex with lxml while writing code for replacing spans with img tags.